Example/FancyShop/Pods/Target Support Files/Pods-FancyShop/Pods-FancyShop-frameworks-Debug-input-files.xcfilelist

```diff
@@ -1,7 +1,7 @@
 ${PODS_ROOT}/Target Support Files/Pods-FancyShop/Pods-FancyShop-frameworks.sh
 ${BUILT_PRODUCTS_DIR}/Alamofire/Alamofire.framework
 ${BUILT_PRODUCTS_DIR}/CocoaLumberjack/CocoaLumberjack.framework
-${PODS_ROOT}/MCSCommerceWeb/MCSCommerceWeb.framework
```

Comment:
Stop tracking pods files.

Reply:
don't think we're tracking them anymore now